src/components/Sum/Sum.jsx

```diff
@@ -1 +1,17 @@
-// export const Sum = () => ();
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    Sum of
+    {' '}
+    {a}
+    {' '}
+    and
+    {' '}
+    {b}
+    {' '}
+    is
+    {' '}
+    {a + b}
```

Comment:
It's hard to read now. Consider using [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for better readability

Reply:
corrected